corpora-api.yml

```diff
@@ -0,0 +1,530 @@
+      parameters:
+        - $ref: '#/components/parameters/query_user_uuid'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  request_id:
+                    type: string
+                  project_uuid:
+                    $ref: '#/components/schemas/project_uuid'
+                  name:
+                    type: string
+                  processing_state:
+                    type: string
+                  validation_state:
+                    type: string
+                  user_uuid:
+                    type: string
+        '400':
+          $ref: '#/components/responses/400'
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+
+    post:
+      summary: Create a new submission
+      tags:
+        - submissions
+      description: >-
+        Opens a new submission with a unique UUID if a submission is not already open for a project. If a project UUID
+        is not provided, then a project UUID will be generated. Otherwise, the project details based on the given
+        project UUID will be used to pre-populate the newly created submission entity. On success, a message will be
+        returned with the project’s UUID.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                needs_attestation:
+                  type: boolean
+                  description: >-
+                    Determines whether this project requires attestation in which case this project cannot be submitted
+                    without a Terms and Conditions file also uploaded.
+                project_uuid:
+                  type: string
+                  description: >-
+                    If the submission is created in order to update an existing publicly published project, then a
+                    Project UUID must be provided in order to backfill the new submission entity with the existing
+                    project details for modification.
+              required:
+                - needs_attestation
+      responses:
+        '201':
+          description: A new submissions has been created.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  project_uuid:
+                    $ref: '#/components/schemas/project_uuid'
+        '400':
+          $ref: '#/components/responses/400'
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+
+
+  /v1/submission/{project_uuid}:
+
+    get:
```

Comment:
Just clarifying as I mentally walk through Data Portal user flows:

- this endpoint would be used to render a "private"/"in review" project
- GET `/v1/project/{project_uuid}` will be used to render a "public" project

Yes?